docs/reference/commandline/plugin_install.md

```diff
@@ -32,20 +32,20 @@ the registry. Note that the minimum required registry version to distribute
 plugins is 2.3.0
 
 
-The following example installs `no-remove` plugin and [set](plugin_set.md) it's env variable
+The following example installs `sample-volume-plugin` plugin and [set](plugin_set.md) it's env variable
 `DEBUG` to 1. Install consists of pulling the plugin from Docker Hub, prompting
 the user to accept the list of privileges that the plugin needs, settings parameters
  and enabling the plugin.
 
 ```bash
-$ docker plugin install tiborvass/no-remove DEBUG=1
+$ docker plugin install tiborvass/sample-volume-plugin DEBUG=1
 
-Plugin "tiborvass/no-remove" is requesting the following privileges:
+Plugin "tiborvass/sample-volume-plugin" is requesting the following privileges:
  - network: [host]
```

Comment:
@there is not privileges in `tiborvass/sample-volume-plugin` it doesn't even ask the question.

Should I use a fake plugin, that doesn't exists ? or mine ?